### PR TITLE
feat(archetype): apply Design System to the archetype

### PR DESCRIPTION
**Pull request by tajespasarela**
EX-3868

### CODE REVIEW

**Comment by tajespasarela** [comment]
The purpose of this PR is to use the Design System in the Archetype to apply styles.

File: src/design-system/tokens.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+:root {
+  --x-color-lead: #36515b;
```
Comment (pmareke) [inline src/design-system/tokens.scss:2]: The default colors are:

```
  --x-color-lead: #36515b;
  --x-color-auxiliary: #667981;
  --x-color-neutral-10: #1a1a1a;
  --x-color-neutral-35: #595959;
  --x-color-neutral-70: #b3b3b3;
  --x-color-neutral-95: #f2f2f2;
  --x-color-neutral-100: #ffffff;
```

So I think you can delete 

- --x-color-lead
-  --x-color-neutral-10
-  --x-color-neutral-35
-  --x-color-neutral-100

**Review by pmareke — CHANGES_REQUESTED** [review]
Great job!

**Review by pmareke — APPROVED** [review]
LGTM

**Review by davidmfempathy — APPROVED** [review]

**Pull request merged**